src/truststore/_api.py

```diff
@@ -85,6 +85,12 @@ def load_verify_locations(
     def __getattr__(self, name: str) -> Any:
         return getattr(self._ctx, name)
 
+    def __setattr__(self, name: str, value: Any) -> None:
```

Comment:
Is there any situation where we don't want to forward `__setattr__` to the inner `_ctx` aside from `verify_mode`? Almost feel like we should reverse our strategy in `__setattr__` here to default to `setattr(self._ctx, name, value)`.

Reply:
I tried it that way first but it caused infinite recursion when setting `_ctx` itself, and when I wrote the conditional to only exclude `_ctx` there were still a few tests failing. Worth some more thought.